src/components/toolbar/toolbar.js

```diff
@@ -115,7 +116,12 @@ const Toolbar = ({
 
     return filterValues
       .filter(({ value }) => typeof value === 'string')
-      .map(({ name, value }) => t('curiosity-toolbar.label', { context: [name, (value === '' && 'none') || value] }));
+      .map(
+        ({ name, value }) =>
+          (isLocale && t('curiosity-toolbar.label', { context: [name, (value === '' && 'none') || value] })) ||
+          (value === '' && 'none') ||
```

Comment:
optional: it comes across as ignoring the `isLocale` check, but it may be good to use a locale string for `none`... as it is currently, this would be one of the few display strings we hardcode

Reply:
we ended up having to allow optional level configuration instead of having a general condition check. there's a subtle change that's highlighted in the screenshot where the `aws` is displayed instead of the full locale string